portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,204 @@
+
+*/
+
+async function getData (){
+    await fetch('https://api.github.com/users/MohamedShatry/repos')
+            .then(data => data.json())
+            .then(res => {
+                let varHeader = SetHeader(res[0].owner);
+                injectHeader(varHeader);
+                res.forEach(repo => {
+                    var returnRepo = SetRepos(repo);
+                    injectRepo(returnRepo);
+                });
+            })
+            .catch(err => {
+                console.log("Error", err);
+                injectError();
+            }); 
+}
+
+
+function SetHeader(repo){
+    var header = {
+        title : repo.login,
+        url: repo.html_url,
+        avatar_url: repo.avatar_url
+    }
+
+    return header;
+}
+
+function SetRepos(repo){
+    repo_content = {
+        title: repo.name,
+        url: repo.html_url,
+        language: repo.language,
+        description: repo.description,
+        web: repo.homepage
+    }
+
+    return repo_content;
+}
+
+/*
+	*Gets data from {header} object and creates elements to render to html
+    *Resulting html looks like this: 
+
+    	<div id="git-header">
+        	<a href={{ header.url }} target="_blank">
+            	<img class="avatar" src= {{ header.avatar_url }}>
+            </a>
+            <b class="git-header-title"> {{ header.title }} </b>
+        </div>
+
+*/
+function injectHeader(header){
+    const gitHeader = document.getElementById("git-header");
+	
+    const avatar_link = document.createElement("a");
+    avatar_link.href = header.url;
+    avatar_link.target = "_blank";
+
+    const avatar = document.createElement("img");
+    avatar.classList.add("avatar");
+    avatar.src = header.avatar_url;
+    
+	avatar_link.appendChild(avatar);
+
+    const title_tag = document.createElement("span");
+    title_tag.classList.add("git-header-title")
+    title_tag.classList.add("bold")
+    const title = document.createTextNode(header.title);
+    title_tag.appendChild(title);
+    
+    gitHeader.appendChild(avatar_link);
+    gitHeader.appendChild(title_tag);
+
+}
+
+/*
+	*Gets data from {repo} object and creates elements to render to html
+    *Resulting html looks like this: 
+	
+        <div class="repo-content">
+        	<p class="repo-header">{{ repo.title }}</p>
+            <p> {{ repo.description }} </p>
+            <div class="lowest-div">
+            	<p class="lowest-tag"> {{repo.language}} </p>
+                <a href= {{ repo.url }} target="_blank">
+                	<p class="lowest-tag arrow-link">View Code→</p>
+                </a>
+
+                <!-- Only rendered if repo has a web url-->
+                <a href= {{ repo.web }} target="_blank">
+                	<p class="lowest-tag arrow-link">View Live→</p>
+                </a>
+
+            </div>
+        </div>
+
+*/
+function injectRepo(repo){
+    const main = document.getElementById("repo-container");
+
+	//Create container to hold content parsed in
```

Comment:
nit:indentation

Reply:
I'm not really sure why the indentation is off. This is how it's formatted on my editor: 
![Screenshot 2020-05-29 at 5 13 46 PM - Display 2](https://user-images.githubusercontent.com/39087499/83306055-e0339300-a1cf-11ea-97cf-aca5d0629363.png)

I'll try to mess around with the settings.